### PR TITLE
add name and symbol to the asset_data table

### PR DESCRIPTION
**Pull request by nikhil-helius**

### CODE REVIEW

**Comment by NicolasPennie** [comment]
Build is failing

File: digital_asset_types/src/dapi/common/asset.rs [hunk 1]
```diff
@@ -151,6 +151,35 @@ pub fn safe_select<'a>(
...
+async fn process_raw_fields(
+    name: Option<String>,
+    symbol: Option<String>,
+    asset_data: &AssetData,
+) -> (Option<String>, Option<String>) {
+    let name_result = if let Some(name) = &name {
+        let raw_name = asset_data.raw_name.clone();
+        match String::from_utf8(raw_name) {
+            Ok(s) => Some(s),
+            Err(_) => Some(name.clone()),
```
Comment (NicolasPennie) [inline digital_asset_types/src/dapi/common/asset.rs:164]: Under what scenarios would this happen? Do you have an example NFT?

File: digital_asset_types/src/dapi/common/asset.rs [hunk 2]
```diff
@@ -151,6 +151,35 @@ pub fn safe_select<'a>(
         .and_then(|v| v.pop())
 }
 
+#[cfg(feature = "fetch-raw-fields")]
+async fn process_raw_fields(
```
Comment (NicolasPennie) [inline digital_asset_types/src/dapi/common/asset.rs:155]: How will this function be used?

Comment (nikhil-helius) [reply digital_asset_types/src/dapi/common/asset.rs:155]: Right now, I am using it for testing purpose. Its disabled due the compile/build time flag. However, based on a new optional field in the api, we will extract the raw fields or the sanitised fields

File: digital_asset_types/src/dapi/common/asset.rs (hunk 1 above)
Comment (nikhil-helius) [reply digital_asset_types/src/dapi/common/asset.rs:164]: It can fail if the utf8 check validation of the string fails. I don't have an example nft

**Comment by nikhil-helius** [comment]
> Build is failing

Test was failing. Fixed it.

**Pull request merged**